index.html

```diff
@@ -920,5 +920,69 @@ <h3>
               </dd>
             </dl>
 </section>
+<section>
+    <h3>
+      {{RTCDataChannel}} extensions
+    </h3>
+    <p>This document extends {{RTCDataChannel}} by making it <a data-cite="!HTML/#transferable-objects">transferable</a>.</p>
+      This allows sending and receiving messages outside the context the connection was created, for instance in workers or third-party iframes.</p>
+    <div>
+      <p>The WebIDL changes are the following:
+      <pre class="idl"
+>[Exposed=(Window,Worker), Transferable]
+partial interface RTCDataChannel {
+};</pre>
+    </div>
+    <div>
+      <p>The <a data-cite="WEBRTC#dfn-create-an-rtcdatachannel">create an RTCDataChannel</a> algorithm is updated by adding the following steps after step 4 of the original algorithm:</p>
+      <ol>
+        <li><p>Initialize <var>channel</var>.`[[IsTransferable]]` to <code>true</code>.</p></li>
+        <li><p>Queue a task to run the following step:</p>
+          <ol>
+            <li><p>Set <var>channel</var>.`[[IsTransferable]]` to <code>false</code>.</p></li>
+          </ol>
```

Comment:
Suggest adding a paragraph saying "This task must be guaranteed to run before any incoming data is presented to "channel"." It is not obvious that the implementation has to enforce that property; in theory there could be ondata events queued behind the onopen event before the onopen event gets started; this transfer algorithm has to ban that implementation strategy.

Reply:
Sounds fair, I'll add something